scripts/ec2/stop.coffee

```diff
@@ -2,58 +2,62 @@
 #   Stop ec2 instance
 #
 # Commands:
-#   hubot ec2 stop --instance_id=[instance_id] - Stop the Instance
+#   hubot ec2 stop <instance_id> [<instance_id> ...] Stop the instances
 #
 # Notes:
-#   --instance_id=***   : [required] One instance ID.
+#   instance_id : [required] The ID of one or more instances to stop. For example, i-0acec691.
 #   --dry-run           : [optional] Checks whether the api request is right. Recommend to set before applying to real asset.
 
+ec2 = require('../../ec2.coffee')
+restrictor = require './restrictor'
+util = require 'util'
+
 getArgParams = (arg) ->
   dry_run = if arg.match(/--dry-run/) then true else false
+  return {dry_run: dry_run}
+
+stopInstances = (msg, params, instances, err) ->
+  return (err) ->
+    if err
+      msg.send "Error! #{err}"
+      return
```

Comment:
Doesn't really matter but these one-liner `if` statements can be condensed into:

```
return msg.send "Error! #{err}" if err
```